README.md

```diff
@@ -31,6 +31,26 @@ Here is a brief list of the available widgets:
 * **edit_box**: an editable text box (multi-line edit_boxes are not yet fully supported).
 * **scroll_frame**: a frame that has scrollable content.
 
+# Getting started 
+```
+git clone https://github.com/unnamed-mmo/lxgui
+sudo dnf install -y freetype-devel SFML-devel lua-bit32 mesa-libGLU-devel lua-devel
```

Comment:
I have never heard of `dnf`! If `dnf` is specific to a particular distro, it may be good to highlight this and add commands for more commonplace distros and package managers (`apt`, `yum`, `vcpkg`, etc.).

For a "quick getting started" section, I would suggest sticking to using the SFML backend, which means the only required packages in this list should be `SFML-devel` and `lua-devel` (I assume `devel` packages will automatically install the right binaries as dependencies in `dnf`).

For `apt` on Ubuntu this would be:
`sudo apt install libsfml-dev liblua5.2-dev`

Reply:
I've added the different distro installs! I'm on Fedora so I was just updating it with what I knew worked lol

Reply:
Thanks!